### DISCUSSION TITLE
Scheduler doesn't apply php ini config to tasks

### DISCUSSION THREAD

**Discussion by brokerUA** [laravel/framework#40705] [Q&A]
Hello, 
When run scheduler:
`/usr/local/php72/bin/php -c /home/server/.system/php/site.ini /home/path/to/artisan schedule:run`
but tasks execute without my ini configuration (added custom php extension).

If run task direct:
`/usr/local/php72/bin/php -c /home/server/.system/php/site.ini /home/path/to/artisan myTask`
ini configuration apply successfully.

**Why custom config does not delegated to tasks?**

Temporary solution run task through `exec` like this:
`$path = '/usr/local/php72/bin/php -c /home/server/.system/php/site.ini /home/path/to/artisan ';`
`$schedule->exec($path . 'myTask')->everyTenMinutes();`

I don’t understand if this is a scheduler error that it does not pick up the config from the request or normal behavior...

_Laravel 7.30.6, PHP 7.2, CentOS 7, Apache_

**Comment by ousid** [comment]
Hey, @brokerUA 

Take a look [here](https://laravel.com/docs/8.x/scheduling#running-the-scheduler), this may help you out.

**Reply by brokerUA** [reply]
Thanks, but in documentation I hasn't find answer on my question. 

On the server different php configs. For apply those I use `-c` flag in cron scheduler. But config dont apply to tasks when run `schedule:run`
